_sass/vendor/OneLightJekyll/syntax.scss

```diff
@@ -1,51 +1,53 @@
+// Generated with OneLightJekyll applied to Atom's One Light theme
```

Comment:
The diff claims:
```
...OneDarkJekyll/syntax-firewatch-green.scss → _sass/vendor/OneLightJekyll/syntax.scss
```
That seems incompatible with the comment. I haven't tried check the color codes.

Reply:
Gotcha. The comment is correct; I don't manually control the git diff and how it chooses to add/rename files (my understanding is that GitHub uses the default git engine to determine the "files changed" tab). Let me see if I can do some hacky commits to override this history.

Reply:
Okay, after doing some local playing around, it'll be *very* hard to simultaneously remove the old dark themes + create this new one, without git marking it as a rename-and-edit; the contents of the file are too similar.

I'd prefer to make this simpler; I think we can just merge this PR first (squashing the commits, etc.) and then immediately follow-up with another PR to remove the other syntax themes. What are your thoughts?

Reply:
LGTM!